src/wombats/components/pedestal.clj

```diff
@@ -31,22 +39,88 @@
 (defn- is-dev?
   "Determins if the service is running in a dev env."
   [service-map]
-  (= (:env service-map) :dev))
+  (= (::env service-map) :dev))
+
+(def log-request
+  "Log the request's method and uri."
+  (interceptor/on-request
+    ::log-request
+    (fn [request]
+      (log/info (format "%s %s"
+                        (clojure.string/upper-case (name (:request-method request)))
+                        (:uri request)))
+      request)))
+
+(defn- default-interceptors
+  "Modified version of
+  https://github.com/pedestal/pedestal/blob/master/service/src/io/pedestal/http.clj#L180
+
+  Adds interceptors to all environments"
+  [service-map]
+  (let [{interceptors ::http/interceptors
+         routes ::http/routes
+         router ::http/router
+         file-path ::http/file-path
+         resource-path ::http/resource-path
+         method-param-name ::http/method-param-name
+         allowed-origins ::http/allowed-origins
+         not-found-interceptor ::http/not-found-interceptor
+         ext-mime-types ::http/mime-types
+         enable-session ::http/enable-session
+         enable-csrf ::http/enable-csrf
+         secure-headers ::http/secure-headers
+         :or {router :map-tree
+              method-param-name :_method
+              ext-mime-types {}
+              enable-session nil
+              enable-csrf nil
+              secure-headers {}}} service-map
+        processed-routes (cond
+                           (satisfies? route/ExpandableRoutes routes) (route/expand-routes routes)
```

Comment:
So this is actually a modified version of pedestal's `default-routes` function. I'm more inclined to want to remain close to the original code here, however I do agree, it looks like it could be cleaned up a bit